tiledb/tests/test_dask.py

```diff
@@ -42,6 +42,7 @@ def _make_multiattr_2d(self, uri, shape=(0, 100), tile=10):
 
         tiledb.DenseArray.create(uri, schema)
 
+    @pytest.mark.flaky(reruns=3, reruns_delay=2)
```

Comment:
Should we use `exclude=[TileDBError]` here?

Reply:
I added [only-rerun](https://pytest-rerunfailures.readthedocs.io/latest/mark.html#only-rerun) argument.